src/modeler/AutoRest.Swagger/Validation/TrackedResourcePatchOperationValidation.cs

```diff
@@ -46,8 +46,8 @@ public override IEnumerable<ValidationMessage> GetValidationMessages(Dictionary<
             IEnumerable<Operation> patchOperations = ValidationUtilities.GetOperationsByRequestMethod("patch", serviceDefinition);
 
             // enumerate all the models returned by all PATCH operations (200/201 responses)
-            var respModels = patchOperations.Select(op => op.Responses["200"]?.Schema?.Reference?.StripDefinitionPath());
-            respModels.Union(patchOperations.Select(op => op.Responses["201"]?.Schema?.Reference?.StripDefinitionPath())).Where(modelName=>!string.IsNullOrEmpty(modelName));
+            var respModels = patchOperations.Select(op => op.Responses?["200"]?.Schema?.Reference?.StripDefinitionPath());
+            respModels.Union(patchOperations.Select(op => op.Responses?["201"]?.Schema?.Reference?.StripDefinitionPath())).Where(modelName=>!string.IsNullOrEmpty(modelName));
```

Comment:
How did you find this issue, just curious since we can start running the rest of our validation rules to check for errors likewise